### PR TITLE
[Encode] Set correct entrypoint for lowpower

### PR DESCRIPTION
**Pull request by Tianhaol**
Set correct entrypoint to VAEntrypointEncSliceLP for HEVC lowpower
insead of VAEntrypointEncSlice

### CODE REVIEW

**Review by thomasli21801 — APPROVED** [review]

File: encode/hevcencode.c [hunk 1]
```diff
@@ -2031,6 +2031,12 @@ static int init_va(void)
     for (i = 0; i < VAConfigAttribTypeMax; i++)
         attrib[i].type = i;
 
+    if (lowpower)
+    {
+        entryPoint = VAEntrypointEncSliceLP;
+        LCU_SIZE = 64;
```
Comment (XinfengZhang) [inline encode/hevcencode.c:2037]: why not use query function to set the LCU_SIZE

Comment (Tianhaol) [reply encode/hevcencode.c:2037]: This is a sample AP.   It is simple to set 64.
We can query the size by this key VAConfigAttribEncHEVCBlockSizes, I must deal with the complicated logic for LCU_SIZE as follows.

- If the value is  nonexistent or fail to query,  I must set it to appropriate value.

- There are two values for LCU size,  I set LCU_SIZE with the min or max, or choose one value between them.
  log2_max_coding_tree_block_size_minus3
  log2_min_coding_tree_block_size_minus3

**Review by XinfengZhang — APPROVED** [review]
LGTM, but it maybe only work with intel implementation, because some of the attributes is not queried then set.

**Comment by Tianhaol** [comment]
> LGTM, but it maybe only work with intel implementation, because some of the attributes is not queried then set.
If the queried values is invalid, how to set default value and fall back to reasonable value is thorny.

**Pull request merged**